data/vyrmeid/vyrmeid.txt

```diff
@@ -20,7 +20,7 @@ ship "Vyrmeid"
 	swizzle 0
 	"never disabled"
 	attributes
-		"category" "Fighter"
+		category "Fighter"
```

Comment:
There are also cases of this in "incipias ships.txt", and one in the "Suicide Gun" definition in "remnant jobs". It would be good to do those here, instead of having another PR for the same change.